### PR TITLE
docs: fix spacing 

### LINKED ISSUE #1602: [docs] fix spacing
- [ ] Spacing between headings and `<uxdot-example>`, sample element, status table, and status checklist should be `--rh-space-lg`

- [x] Spacing between any content and the footer should be `--rh-space-5xl` for all pages. Currently looks like there's extra space below the Related elements/Feedback rows.

Screenshot 2024-06-10 at 12 51 41 PM

- [x] Spacing between two headings should be `--rh-space-2xl`. I'm seeing no space between headings on token pages:
Screenshot 2024-06-10 at 12 51 28 PM

- added a spacing edit to #1598, which is in progress already

- [x] Spacing in the homepage cards is a bit off. Seeing too much between the image and the heading. Margins are also getting added to `<p>`. Feel free to tackle this in a separate issue or PR if this is something that needs to be fixed in `<rh-card>`.
Screenshot 2024-06-11 at 10 40 01 AM
Screenshot 2024-06-11 at 10 39 55 AM

@coreyvickery, any more spacing fixes to add?

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1.  Added `--rh-space-lg` margin between header and status list in `uxdot-repo-status-list`
2. Added `--rh-space-lg` margin for `uxdot-example` when a sibling of `uxdot-copy-permalinks` and headers

Closes #1602 

### TODO: 
- [x] Fix spacing between sibling headers on tokens pages

## Testing Instructions

1. View [Deploy Preview](https://deploy-preview-1610--red-hat-design-system.netlify.app/tokens/color/)

### CODE REVIEW

File: docs/assets/javascript/elements/uxdot-repo-status-list.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ class UxdotRepoStatusList extends LitElement {
     #inner-container {
       border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle-on-light, #c7c7c7);
       border-radius: var(--rh-border-radius-default, 3px);
+      margin-block-start: var(--rh-space-lg, 16px);
```
Comment (marionnegp) [inline docs/assets/javascript/elements/uxdot-repo-status-list.js:14]: I'm seeing some extra space below the heading, but I can't figure out where it's coming from. Might be better to use `--rh-space-md, 8px` to make the `margin-block-end` look correct compared to other `<h2>`s

Screenshot 2024-07-18 at 3 15 42 PM

**Review by marionnegp — CHANGES_REQUESTED** [review]
Just one quick change!

File: docs/assets/javascript/elements/uxdot-repo-status-list.js (hunk 1 above)
Comment (zeroedin) [reply docs/assets/javascript/elements/uxdot-repo-status-list.js:14]: image

I am not seeing this myself.

What browser did you use for the above screen shot?

Comment (marionnegp) [reply docs/assets/javascript/elements/uxdot-repo-status-list.js:14]: I was using Chrome, but I don't see it anymore either. I'll go ahead and resolve this.

**Review by marionnegp — APPROVED** [review]

**Pull request merged**